HISTORY.rst

```diff
@@ -17,6 +17,17 @@ History
   * ``paysera``
   * ``payvision``
   * ``trustly``
+* The ``/credit_card/last_4_digits`` input has been deprecated in favor of
+  ``/credit_card/last_digits`` and will be removed in a future release.
+  ``last_digits``/``last_4_digits`` also now supports two digit values in
+  addition to the previous four digit values.
+* Eight digit ``/credit_card/issuer_id_numbers`` inputs are now supported in
+  addition to the previously accepted six digit ``issuer_id_number``s. If you
```

Comment:
GitHub's reStructuredText parser doesn't like the "s" after the backticks. I assume other implementations will also parse it in unintended ways.